app/forms/state_file/intake_login_form.rb

```diff
@@ -11,10 +11,19 @@ def intake
       @intake
     end
 
+    def intake_to_log_in(matching_intakes)
+      with_accepted_submissions = matching_intakes.filter { |intake| intake.latest_submission&.in_state?(:accepted) }
```

Comment:
Would we need to look at all submissions for this? If for w/e reason the client submits and erroneous submission after acceptance or submits more than one and the one accepted is not the chronological last submission, this logic could return an undesired intake or not at all.